### ISSUE TITLE
Create footer utlity for always at bottom- of page and after content

### ISSUE THREAD

**Issue by GregJohnStewart** [twbs/bootstrap#38111]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Proposal

I would like a utility similar to `sticky-footer` that has slightly different behavior, as described here: https://www.freecodecamp.org/news/how-to-keep-your-footer-where-it-belongs-59c6aa05c59c/ 

![image](https://cdn-media-1.freecodecamp.org/images/1*gUfDwoSlbdxjXv10Pxnxtw.png)

If there's currently a way to do this, apologies that I missed it.

If it cannot be summed up into a single utility, might also be helpful to have an example featuring this behavior.

### Motivation and context

This footer behavior is desirable to me.

**Comment by alexandergitter** [comment]
I don't think absolutely positioning the footer like this is the modern way to achieve the "sticky" footer. You can use flexbox/grid to get this to work.
For example via Bootstrap's helper classes:
```html
<body class="min-vh-100 vstack">

  <main class="flex-grow-1">
   main content area
  </main>

  <footer>
    here is the footer
  </footer>

</body>
```

**Comment by GregJohnStewart** [comment]
Thanks a bunch, this worked extremely well for me! Might just say a todo here is to add an example in the docs, but I'll leave it up to you guys

**Comment by julien-deramond** [comment]
Yep @GregJohnStewart, please let it open so that we can think about adding it as an example. Thanks for the idea btw.

**Comment by mdo** [comment]
Not a hard no, but I'm not doing it. Closing as won't fix.
